api/utils/keys/policy.go

```diff
@@ -138,19 +138,28 @@ func GetPolicyFromSet(policies []PrivateKeyPolicy) (PrivateKeyPolicy, error) {
 	return setPolicy, nil
 }
 
-var privateKeyPolicyErrRegex = regexp.MustCompile(`private key policy not met: (\w+)`)
+var (
+	privateKeyPolicyErrRegex = regexp.MustCompile(`private key policy not satisfied: (\w+)`)
```

Comment:
You could solve this with a single regex:

```suggestion
	privateKeyPolicyErrRegex = regexp.MustCompile(`private key policy not (met|satisfied): (\w+)`)
```

Just make sure to update the number of subMatches/policy group index below.